internal/frontend/http/image.go

```diff
@@ -81,6 +81,7 @@ func (f *Frontend) handleImage(ctx context.Context, w http.ResponseWriter, r *ht
 	}
 
 	w.Header().Set("Content-Disposition", fmt.Sprintf(`attachment; filename="%s"`, path))
+	w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
I think this is only part of the story, you need to implement `OPTIONS` first and probably should inject this for all GET APIs. 

And of course it should be covered with tests

Reply:
We don't need it for all GET APIs though